tools/ci_build/build.py

```diff
@@ -551,7 +554,7 @@ def main():
         cmake_extra_args = []
         if(is_windows()):
           if (args.x86):
-            cmake_extra_args = ['-A','Win32','-G', 'Visual Studio 15 2017']
+            cmake_extra_args = ['-A','Win32','-T','host=x64','-G', 'Visual Studio 15 2017']
```

Comment:
Why setting toolchain to x64 even for x86 build?

Reply:
it goes out of heap space with 32 bit toolchain